### ISSUE TITLE
[java] `@implNote` tag  in  javadoc triggers an error message

### ISSUE THREAD

**Issue by v-ghost03** [pmd/pmd#5765]
**Affects PMD Version:** 
7.3.0

**Rule:**
```
error: unknown tag: implNote
 * @implNote The implementation with {@code Map}.
   ^
```

<img width="651" alt="Image" src="https://github.com/user-attachments/assets/ea165759-9e02-42ef-aef5-523fd39ca173" />

**Description:**
 Adding the @implNote tag causes a build error. This should not be an error. 

**Code Sample demonstrating the issue:**

```java
package twosum;

import java.util.HashMap;
import java.util.Map;

/**
 * @implNote The implementation with {@code Map}.
 */
public class TwoSum {
   public int[] twoSum(int[] nums, int target) {
       
        return new int[]{-1, -1};
    }
}
```

**Expected outcome:**

PMD reports a violation at line ..., but that's wrong. That's a false positive.

**Running PMD through:**  Gradle

**Comment by v-ghost03** [comment]
Reproduced on PMD version 7.13.0.

**Comment by v-ghost03** [comment]
UPD: The same for `implSpec` and `apiNote`

**Comment by oowekyala** [comment]
This does not look like a PMD error... It looks more like your javadoc plugin is misconfigured. You have to register custom tags. See [this SO question](https://stackoverflow.com/questions/33246400/gradle-build-javadoc-task-fails-with-unknown-tag-attr).

**Comment by adangel** [comment]
FYI: In PMD, we also use additional tags in javadoc, and our configuration for maven-javadoc-plugin looks like this:

https://github.com/pmd/pmd/blob/d47399067ce9842ecabc526e0c0687a7eb9182ba/pom.xml#L415-L443
